docs/install/install-server/install-server.inc

```diff
@@ -59,36 +59,97 @@ Install the software
 
+        Please note, the path to the virtual env is arbitrary. Your desired path can override below example.
+
+        .. code-block:: sh
+
+            # Install GCC
+            sudo apt-get update
+            sudo apt-get install build-essential
+            sudo apt-get install python3-pip
+
+            # Install python3 >= 3.6, venv, Python pip, wheel and git
+            # pip and pip3 are used interchangeably
+            sudo apt-get install python3-venv
+            sudo python3 -m venv /opt/inmanta
+            sudo /opt/inmanta/bin/pip install wheel
+            sudo /opt/inmanta/bin/pip install inmanta
+            sudo /opt/inmanta/bin/inmanta --help
+            
+            # Install PostgreSQL
+            sudo apt-get install postgresql postgresql-client
+
+
+        Download the configuration file named ``inmanta.cfg`` (this name is arbitrary) in your virtual env:
+
+        .. code-block:: sh
+
+            sudo apt-get install wget
+            sudo wget -O /opt/inmanta/inmanta.cfg "https://raw.githubusercontent.com/inmanta/inmanta-core/master/misc/inmanta.cfg"
+
+
+        If you want to use the dashboard you need to install it as well:
+
+        Get the pre-built package from `our github page <https://github.com/inmanta/inmanta-dashboard/packages/>`_ click on the latest release and then on the right hand side, under ``Assets``, you will see the compressed package. Download and extract it to your desired directory (preferably, on the same virtual env which was created earlier, in this case, /opt/inmanta). Next, open the ``inmanta.cfg`` file and at the bottom of the file, under ``[dashboard]`` section, change the ``path`` value to the ``dist`` directory of where you extracted the pre-built package. For instance:
+
+        .. code-block:: ini
+
+            path=/opt/inmanta/package/dist
+
+
+        Then the dashboard can be started using below command (please note, below command has to be run after completing the  :ref:`configure-server`) part:
+
+        .. code-block:: bash
+
+            inmanta -vv -c /opt/inmanta/inmanta.cfg server
+
+
+    .. tab:: Other
+
+        First make sure Python >= 3.6 and git are installed. Inmanta requires many dependencies so it is recommended to create a virtual env.
+        Next install inmanta with ``pip install`` in the newly created virtual env.
+
+        Please note, the path to the virtual env is arbitrary. Your desired path can override below example.
+
         .. code-block:: sh
 
             # Install python3 >= 3.6 and git
+            # If git is not already installed, by running git in your terminal, the installation guide will be shown
             sudo python3 -m venv /opt/inmanta
             sudo /opt/inmanta/bin/pip install inmanta
             sudo /opt/inmanta/bin/inmanta --help
+            
 
+        Install PostgreSQL using this `guide <https://www.postgresql.org/docs/9.3/tutorial-install.html/>`_
```

Comment:
* For some reason the slash at the end breaks the link.
* PostgreSQL 10 is the official backend for the inmanta server. I am not sure whether PostgreSQL 9 will work.
```suggestion
        Install PostgreSQL using this `guide <https://www.postgresql.org/docs/10/tutorial-install.html>`_
```